mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -398,6 +400,8 @@ CLASS net/minecraft/class_1297 net/minecraft/entity/Entity
 		ARG 2 pos
 	METHOD method_36974 tryCheckBlockCollision ()V
 	METHOD method_36975 playExtinguishSound ()V
+	METHOD method_37215 playAmethystChimeSound (Lnet/minecraft/class_2680;)V
```

Comment:
Need to rename `Entity.prevAge` field to something like `lastChimeAge` because it's exclusively used in this method.